ocp_install_env.sh

```diff
@@ -78,12 +69,25 @@ function generate_ocp_install_config() {
     fi
 
     mkdir -p "${outdir}"
+
+    # IPv6 network config validation
+    if [[ "${EXTERNAL_SUBNET}" =~ .*:.* ]]; then
+      if [[ "${NETWORK_TYPE}" != "OVNKubernetes" ]]; then
+        echo "NETWORK_TYPE must be OVNKubernetes when using IPv6"
```

Comment:
Could we just set this to OVNKubernetes if we detect IPv6?

Reply:
I guess we could - @russellb do you think anyone is likely to test anything other than OVNKubernetes here?

Reply:
They shouldn't.  I'm OK with just setting it instead of telling them to set it ...

Reply:
let's do this in a follow-up